schema.yaml

```diff
@@ -89,28 +89,28 @@ schema;game:
 schema;genres:
   type: str
   enum: [
+    'Action',
```

Comment:
I am against the removal of Horror / Survival Horror as a main genre due to its well-defined and unique gameplay characteristics.